Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInputEditor.cs

```diff
@@ -46,20 +46,22 @@ public void OnEnable()
             #if UNITY_INPUT_SYSTEM_ENABLE_UI
             m_UIInputModuleProperty = serializedObject.FindProperty(nameof(PlayerInput.m_UIInputModule));
             #endif
+
+            m_Data = new PlayerInputEditorAnalytic.Data(target as PlayerInput);
         }
 
-        public void OnValidate()
+        public void OnDisable()
         {
-            ++m_AnalyticsData.onValidateCount;
+            new InputComponentEditorAnalytic(InputSystemComponent.PlayerInput).Send();
+            var data = new PlayerInputEditorAnalytic.Data(target as PlayerInput);
+            if (!data.Equals(m_Data))
```

Comment:
Seems we are only sending this if its not the default, why not always send. Especially if in future the defaults change.

Reply:
To reduce cost since we anyway has the above log being unconditionally logged so we know that e.g. COUNT(defaultSettings) = COUNT(ComponentAnalytic where PlayerInput) - COUNT(PlayerInputComponentAnalytic) is the number of occurrences that would map to default settings.

If defaults change over time, the associated meta data and template docs need updating as well, but it would definitely be easier to always log this both in code and in post processing. Will ask in today's meeting what they recommend. If we go for always logging we can also avoid making the data equatable which removes some code.

Reply:
After follow up with data engineer I will remove these conditional checks

Reply:
This have been adressed now